src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.tsx

```diff
@@ -30,6 +33,20 @@ export const IncidentsDetail = ({
 }: Props) => {
   const { _display, text, location, extra_properties, _links } = incidentsDetail
   const attachments = _links?.['sia:attachments']
+  const [selectedAttachment, setSelectedAttachment] = useState<string | null>(
+    null
+  )
+  const [selectedAttachmentEntity, setSelectedAttachmentEntity] =
+    useState<typeof attachments>()
```

Comment:
There exist a `Attachment` type in /types

Reply:
Yup, but there is a story in this sprint that will take care of all the different places that we define Attachment. It happens at multiple places that we redefine Attachment again. Changing it here is out-of-scope.